samples/excel/30-events/events-comment-event-handler.yaml

```diff
@@ -0,0 +1,195 @@
+                console.log(`    Comment author: ${changedComment.authorName}`);
+                console.log(`    Creation date: ${changedComment.creationDate}`);
+            });
+        }
+
+        async function commentDeleted(event: Excel.CommentDeletedEventArgs) {
+            await Excel.run(async (context) => {
+                // Print out the deleted comment's ID.
+                console.log(`A comment was deleted:`);
+                console.log(`    ID: ${event.commentDetails[0].commentId}`);
+            });
+        }
+
+        async function addCommentToSelectedCell() {
+            await Excel.run(async (context) => {
+                const selectedRange = context.workbook.getSelectedRange();
+
+                // Note that an InvalidArgument error will be thrown if multiple cells are selected.
+                context.workbook.comments.add(selectedRange, "TODO: add headers here.");
+
+                await context.sync();
+            });
+        }
+
+        async function addCommentToCell() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Comments");
+
+                // Note that an InvalidArgument error will be thrown if multiple cells are passed to `comment.add`.
+                sheet.comments.add("A2", "TODO: add data.");
+                await context.sync();
+            });
+        }
+
+        async function editComment() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Comments");
+                const comment = sheet.comments.getItemAt(0);
+                comment.content = "PLEASE add headers here.";
+                await context.sync();
+            });
+        }
+
+        async function deleteComment() {
+            await Excel.run(async (context) => {
+                context.workbook.comments.getItemByCell("Comments!A2").delete();
+                await context.sync();
+            });
+        }
+
+        async function setup() {
+            await Excel.run(async (context) => {
+                context.workbook.worksheets.getItemOrNullObject("Comments").delete();
+                const sheet = context.workbook.worksheets.add("Comments");
+
+                sheet.activate();
+                await context.sync();
+            });
+        }
+
+        /** Default helper for invoking an action and handling errors. */ 
+        async function tryCatch(callback) {
+            try {
+                await callback();
+            } catch (error) {
+                // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+                console.error(error);
+            }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p>This sample shows how to add, edit, and remove comments.</p>
```

Comment:
Please update this comment to reflect the sample's goal of showing comment-related events.